src/main/java/appeng/block/crafting/CraftingStorageBlock.java

```diff
@@ -24,7 +24,7 @@
 
 public class CraftingStorageBlock extends AbstractCraftingUnitBlock<CraftingStorageBlockEntity> {
```

Comment:
This class can probable be removed.

Reply:
If it comes down to removing CraftingStorageBlockEntity as above then this too shall go.